packages/manager/modules/pci/src/projects/project/storages/containers/add/add.controller.js

```diff
@@ -30,8 +31,10 @@ export default class PciStoragesContainersAddController {
     this.PciProjectStorageContainersService = PciProjectStorageContainersService;
     this.storagePricesLink =
       STORAGE_PRICES_LINK[ovhSubsidiary] || STORAGE_PRICES_LINK.DEFAULT;
+
     this.OBJECT_CONTAINER_NAME_PATTERN = OBJECT_CONTAINER_NAME_PATTERN;
     this.OBJECT_CONTAINER_OFFERS = OBJECT_CONTAINER_OFFERS;
+    this.OBJECT_CONTAINER_OFFERS_LABELS = OBJECT_CONTAINER_OFFERS_LABELS;
```

Comment:
Why import when it is not used in this controller ?

Reply:
Used into HTML template :)